src/scheduler/mod.rs

```diff
@@ -49,6 +51,9 @@ impl SchedulerInput {
 		}
 	}
 }
+
+#[cfg_attr(target_arch = "x86_64", repr(align(128)))]
+#[cfg_attr(not(target_arch = "x86_64"), repr(align(64)))]
```

Comment:
As before. I think the size is wrong and reusing the constant would make it easier to maintain and understand

Reply:
See above